api/v1beta1/nutanixcluster_types.go

```diff
@@ -96,6 +117,7 @@ type NutanixClusterStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:printcolumn:name="ControlplaneEndpoint",type="string",JSONPath=".spec.controlPlaneEndpoint.host",description="ControlplaneEndpoint"
 // +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.ready",description="in ready status"
+// +kubebuilder:printcolumn:name="FailureDomains",type="string",JSONPath=".status.failureDomains",description="NutanixCluster FailureDomains"
```

Comment:
this is a list of failure domains and not a string.

Reply:
Yes, it is map type and shows like. I think it's useful for the users to see the cluster's configured/validated failureDomains without its maniests.

NAME                                                     CONTROLPLANEENDPOINT   READY   FAILUREDOMAINS
nutanixcluster.infrastructure.cluster.x-k8s.io/capx-y5   10.23.131.101          true    {"fd-t1":{"controlPlane":true},"fd-t11":{"controlPlane":true},"fd-t2":{"controlPlane":true},"fd-t22":{"controlPlane":true}}